### ISSUE TITLE
install and manage core on disconnected environments

### ISSUE THREAD

**Issue by johandanforth** [aspnet/Home#1263]
We're disconnected from Internet, no direct access allowed for security reasons. We have local nuget repo and such. Is there any information or guides on how to install and support .net and asp.net core in such environments?

**Comment by tugberkugurlu** [comment]
Sync the nuget.org packages to your local NuGet server?

**Comment by johandanforth** [comment]
But which packages? How to get started? Are there any offline installers for .net core to begin with?

**Comment by tugberkugurlu** [comment]
I am not aware of any offical ways, I am just thinking how you can unblock yourself for now.

This is the list of packages for nightlies (basically unstable packages): https://www.myget.org/gallery/aspnetvnext You can get the list from this feed and pull the corresponding stable ones from nuget.org. This may give you an idea: http://weblogs.asp.net/jongalloway/downloading-a-local-nuget-repository-with-powershell

**Comment by johandanforth** [comment]
Setting up a local nuget repo is one thing but how can one even install dnvm etc and have them pointing at local servers or folders for listing and installing frameworks? I'm sure I'm not the only one who wants to start using .net core on secure networks.

**Comment by tugberkugurlu** [comment]
dnvm is two files in windows: one cmd and one ps1. Copy them somewhere and add that to ur path. Then, change the feed to your local nuget feed theough the DNX_FEED env variable.

This will probably change with dotnet cli, in fact there is aleady an msi for donet cli.

**Comment by NatanelMaman** [comment]
im facing with the same issue.....
is there any instruction ?

**Comment by aspnet-hello** [comment]
This issue is being closed because it has not been updated in 3 months.

We apologize if this causes any inconvenience. We ask that if you are still encountering this issue, please log a new issue with updated information and we will investigate.
